tensorflow/compiler/xla/service/gpu/matmul_utils.cc

```diff
@@ -819,6 +819,10 @@ StatusOr<se::cuda::BlasLt::Epilogue> AsBlasLtEpilogue(
   TF_ASSIGN_OR_RETURN(
       se::blas::ComputationType computation_type,
       GetBlasComputationType(output_layout.dtype, config.compute_precision));
+  if (lhs_layout.dtype == F8E4M3FN || lhs_layout.dtype == F8E5M2) {
+    computation_type = se::blas::ComputationType::kF32;
```

Comment:
Instead of overwriting computation_type after it has been set by GetBlasComputationType, can that function be modified to return the right value?

Reply:
+1, you'll have to pass the lhs dtype into GetBlasComputationType but this is fine.

Can you add a test as well? How were matmuls with FP8 inputs and non-FP8 outputs ever able to work without this PR?